dashboard/config/locales/en.yml

```diff
@@ -23,6 +23,7 @@ en:
   errors:
     messages:
       not_saved: 'Please correct the following errors:'
+      too_young: 'You must be at least 13 to access this content.'
```

Comment:
I seem to remember something about not being allowed to tell people the exact number